pages/activities.jsx

```diff
@@ -154,6 +154,15 @@ var ActivitiesPage = React.createClass({
               description="These six hands-on activities engage learners in thinking critically about online privacy by creating secure passwords, understanding how and where their data is being collected, and more. This is an ideal kit for Mozilla Clubs."
             />
             <ActivityKit
+              src1x="https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png"
+              src2x="https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png"
```

Comment:
if we don't have a retina version of image file than we don't have to explicitly pass in `src2x` :bowtie: